SignalServiceKit/src/Devices/OWSReadReceipt.h

```diff
@@ -1,23 +0,0 @@
-//  Copyright © 2016 Open Whisper Systems. All rights reserved.
-
-#import "TSYapDatabaseObject.h"
-
-NS_ASSUME_NONNULL_BEGIN
-
-@class YapDatabase;
-
-@interface OWSReadReceipt : TSYapDatabaseObject
-
-@property (nonatomic, readonly) NSString *senderId;
-@property (nonatomic, readonly) uint64_t timestamp;
-@property (nonatomic, readonly, getter=isValid) BOOL valid;
```

Comment:
I also removed the "is valid" logic in this class.  We detect and ignore invalid read receipts in the read receipts manager.